Source/WebCore/rendering/PaintPhase.h

```diff
@@ -60,7 +60,7 @@ enum class PaintBehavior : uint32_t {
     SkipSelectionHighlight              = 1 << 1,
     ForceBlackText                      = 1 << 2,
     ForceWhiteText                      = 1 << 3,
-    RenderingSVGMask                    = 1 << 4,
+    RenderingSVGClipOrMask              = 1 << 4,
```

Comment:
It's ok, but technically incorrect for legacy (where it is only used for masks).